Formula/libgr.rb

```diff
@@ -12,9 +12,16 @@ class Libgr < Formula
 
   depends_on :xcode => :build
   depends_on "cairo"
+  depends_on "ffmpeg"
+  depends_on "glfw"
+  depends_on "qt"
+  depends_on "zeromq"
 
   def install
-    system "make", "self", "GRDIR=#{prefix}"
+    # TODO: Remove "-j1" when released archive includes
+    # the fix of https://github.com/sciapp/gr/pull/101 .
+    system "make", "GRDIR=#{prefix}", "-j1"
```

Comment:
If really necessary use `ENV.deparallelize`.

Reply:
Thanks.
I didn't know it. I've changed.